### PR TITLE
Halloween Mad Lib

### PR DESCRIPTION
**Pull request by nathalienordstrom**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:
I didn´t manage to do the dropdown select. It just didn't show in the mad lib. so I put a radio button that one, and I will continue to try to work on de the drop down.

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,449 @@
 body {
```
Comment (annahasselstrom) [inline code/src/index.css:1]: There is indentation and extra lines throughout your css that needs to be fixed. And maybe remove the code that you are not using for example the code referring to checkboxes?
 I really appreciate when you add your comments on your cards - that helps me to understand what your code does! 🙌

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,449 @@
...
+		width: 200px;
+		height: 200px;
+  }
+  
+  button:focus, button:hover {
+    width: 200px;
+    height: 200px;
+  }
 }
+@media (min-width: 1024px) {
```
Comment (annahasselstrom) [inline code/src/index.css:387]: Nice with the responsiveness! 🤩

File: code/src/index.js [hunk 3]
```diff
@@ -1,6 +1,513 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import { App } from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+
```
Comment (annahasselstrom) [inline code/src/index.js:5]: As with your css, you need to go over your indentation and extra empty lines. Like I commented earlier, I find it a little difficult to follow your code - maybe next time you can practice breaking functionality out in their own components?

**Review by annahasselstrom — CHANGES_REQUESTED** [review]
Wow Nathalie! What an ambitious project! A great idea both in execution and styling. You are really enjoying the styling bit - I can tell! Keep it up! It is so much fun and playful. And your styling really fits your theme! Well done 🥳.
In order to reach blue level (I think) you need to add a select and a summary. The summary should hide the previous questions when clicked on. You might also need to look at the accessibility - the screenreader I used did not read out the questions and also I could not tab on everything. I know radiobuttons are super-difficult to make accessible but maybe add some aria-attributes and labels. So there are a couple of things to add still, BUT I love your idea and your styling! 👏    😍

File: code/src/index.js [hunk 4]
```diff
@@ -1,6 +1,513 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ReactDOM from 'react-dom'
```
Comment (EmOlsz) [inline code/src/index.js:2]: There is a convention which says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, `style.css` should be separated as well and located below local dependencies. Ideally, it should look like this :

```jsx
import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';
```

So, final order is :

1. Outer dependencies
2. Local dependencies
3. Style file(s)

File: code/src/index.js [hunk 5]
```diff
@@ -1,6 +1,513 @@
...
+  const handleSubmit = event => {
+    event.preventDefault();
+    setSummary(true);
+  };
+  return (
+
+
+    <div className="container">
+      <div className="header"><h1>Time For Halloween<br>
+      </br> Mad Lib</h1></div>
```
Comment (EmOlsz) [inline code/src/index.js:44]: We shouldn't use `br` tags at all, you can achieve same result by proper CSS styles (display property perhaps?)

File: code/src/index.js [hunk 6]
```diff
@@ -1,6 +1,513 @@
...
+                  <br></br>
+                  <label>
+                    <input
+                      type="checkbox"
+                      value="Maksymilian"
+                      onChange={event => setNameOne(event.target.value)}
+                      checked={nameOne === "Maksymilian"}
+
+                    />
+            Maksymilian<br></br>
```
Comment (EmOlsz) [inline code/src/index.js:131]: Great setup ;)

**Review by EmOlsz — COMMENTED** [review]
Hey Nathalie! I have some mixed feelings about your project... You did great job when it comes to React logic, and that's the most important part since it was React project.
On the other hand, you haven't used any components, all of your code is inside App.js, this needs to be fixed. Same goes with `br` tags, they are super old and shouldn't be used by us at all.
My mixed feelings are about that you did great job with advanced parts of this week project, but forgot about basics (components with props and `br` tags). I'm 100% sure you know how to do that :)
And with that long comment, let's do it. I'll wait for your changes and only then I'll approve whole project

**Comment by nathalienordstrom** [comment]
Hey! I think I fixed everything you asked for. Look and see and get back to me if not. Thank you

**Comment by EmOlsz** [comment]
Hey Nathalie! What I meant by dividing code to separate components, was about slicing your code to smaller chunks, so instead of :

```
const App = () => {
  return (
    <div>
      // Some code 1...
    </div>
    <div>
      // Some code 3...
    </div>
    <div>
      // Some code 3...
    </div>
  )
}
```

you would have :

```
const App = () => {
  return (
    <ComponentOne someProp={someData} />
    <ComponentTwo someProp={someData} />
    <ComponentThree someProp={someData} />
  )
}
```

Please, fix that one as well :)

**Comment by nathalienordstrom** [comment]
okey so i was not suppos to move everything like I did?

**Comment by EmOlsz** [comment]
Yes, not really :)

**Comment by nathalienordstrom** [comment]
I think I got it now!

**Comment by EmOlsz** [comment]
Exactly, it's ok now :)

**Pull request closed without merging**